pkg/daemon/rpm-ostree.go

```diff
@@ -96,7 +97,47 @@ func (r *RpmOstreeClient) GetBootedDeployment() (*RpmOstreeDeployment, error) {
 		return nil, errors.Wrapf(err, "failed to parse `rpm-ostree status --json` output (%s)", truncate(string(output), 30))
 	}
 
-	for _, deployment := range rosState.Deployments {
+	return &rosState, nil
+}
+
+func (r *RpmOstreeClient) Initialize() error {
+	// This replicates https://github.com/coreos/rpm-ostree/pull/2945
```

Comment:
should this comment be for the whole func?

Reply:
Mmm...I think it's right as is because we're just doing the `systemctl start` bit, but not the "dump status on error" bit which is instead in the daemon code.  But it'd probably make sense to live here too....done!